File: uri_exercicios_em_Cpp/beginner/event_time.cpp
#include <iostream>

using namespace std;

void show_rslt(int rslt){
    cout << rslt/3600/24 << " dia(s)" << endl;
    cout << rslt/3600%24 << " hora(s)" << endl;
    cout << rslt%3600/60 << " minuto(s)" << endl;
    cout << rslt%3600%60 << " segundo(s)" << endl;
}

int get_rslt(){

    int rslt {0};
    int d{},h{},m{},s{};
    char var[3];

    for (int i = 0 ; i < 2; i ++){
        cin >> var >> d;
        cin >> h >> var >> m >> var >> s;

        if(rslt == 0)
          rslt = (d*24 + h)*3600 + m*60 + s;

        else
          rslt = (d*24 + h)*3600 + m*60 + s - rslt;
    }


    if(rslt <= 0) rslt += 24*3600;

    return rslt;
}


int main(){

    int rslt = get_rslt();

    show_rslt(rslt);

    return 0;
}

File: beecrowd_exercicios_em_python/begginner/event_time.py
def show(x):
    print("{} dia(s)\n"
          "{} hora(s)\n"
          "{} minuto(s)\n"
          "{} segundo(s)\n".format(
            x // 3600 // 24,
            x // 3600 % 24,
            x % 3600 // 60,
            x % 3600 % 60
                        )
          )


def get_seconds():
    a = input().split()
    b = input().split(" : ")

    return (int(a[1]) * 24 + int(b[0])) * 3600 + int(b[1]) * 60 + int(b[2])


def solve():
    rslt = get_seconds()
    rslt = get_seconds() - rslt

    if rslt <= 0:
        rslt += 24 * 3600

    show(rslt)


solve()
